README.md

```diff
@@ -57,7 +57,7 @@ Run a local instance of Starknet Devnet
 optional arguments:
   -h, --help            show this help message and exit
   -v, --version         Print the version
-  --host HOST           Specify the address to listen at; defaults to localhost (use the address the program outputs on start)
+  --host HOST           Specify the address to listen at; defaults to 127.0.0.1 (use the address the program outputs on start)
   --port PORT, -p PORT  Specify the port to listen at; defaults to 5000
```

Comment:
Can you change this to 5050?

Reply:
[e02385f](https://github.com/Shard-Labs/starknet-devnet/pull/76/commits/e02385f4e597f52983240fdcbd59001b1147deaf)